### PR TITLE
docs: clearify contribution

### PR DESCRIPTION
**Pull request by haxxmaxx**
moving up the contribution section in README so you wont miss it. Also some clarification around releasing

### CODE REVIEW

File: .github/CONTRIBUTION.md [hunk 1]
```diff
@@ -64,19 +66,20 @@ You can skip git commit hooks with:
...
+- Pushing the release commit and tag to main
 
 ### Step-By-Step
 
-1. Check out master and run `git pull`.
-1. Run `git clean -dfx && yarn` to make sure depenencies are up-to-date.
-1. Run `npm version [major | minor | patch] -m "chore(release): v%s"`. Use semver string based on conventional commits since last release. Ex: `npm version patch -m "chore(release): v%s"`.
+1. Check out main and run `git pull`.
+1. Run `git clean -dfx && yarn` to make sure dependencies are up-to-date.
+1. Run `npm version [major | minor | patch] -m "chore(release): v%s"`. Use a [semVer string based on conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) since last release. Ex: `npm version patch -m "chore(release): v%s"`.
```
Comment (Donya-qlik) [inline .github/CONTRIBUTION.md:82]: You might wanna add a hint about major/minor... if you think it helps

**Review by Donya-qlik — APPROVED** [review]
lgtm

File: .github/CONTRIBUTION.md (hunk 1 above)
Comment (haxxmaxx) [reply .github/CONTRIBUTION.md:82]: that is what the link is for

Comment (haxxmaxx) [reply .github/CONTRIBUTION.md:82]: It is basically standard practice, so I don't think we have to explain it in our docs when it is already well documented

**Pull request merged**